test/xpu_api/numerics/complex.number/complex.member.ops/assignment_complex.pass.cpp

```diff
@@ -20,14 +20,14 @@ test()
     dpl::complex<T> c;
     assert(c.real() == 0);
     assert(c.imag() == 0);
-    dpl::complex<T> c2(1.5, 2.5);
+    dpl::complex<T> c2(1.5f, 2.5f);
```

Comment:
Here and in all text places we use postfix ```f``` to create float const instead of ```double``` const.